### PR TITLE
Add oc target

### PR DESCRIPTION
**Pull request by tnozicka**
/cc @soltysh

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -35,6 +35,11 @@ RPM_EXTRAFLAGS := \
 
 IMAGE_REGISTRY :=registry.svc.ci.openshift.org
 
+oc: GO_BUILD_PACKAGES:=./cmd/oc
```
Comment (soltysh) [inline Makefile]: I like the idea, although I wonder if we want to introduce such granular targets. None of the other repos have it, I don't want oc to be the raisin :wink: 
/hold

Comment (tnozicka) [reply Makefile]: I think the one most useful target could be justified, I don't feel strong

**Comment by soltysh** [comment]
/lgtm cancel
@tnozicka how about alternative solution from #51

**Comment by soltysh** [comment]
Closing due to age. 
/close

**Pull request closed without merging**